Author: Diane Trout <diane@ghic.org>
Description: Point to local favicon.ico file instead of referencing
 public url
Forwarded: not-needed.

--- a/distributed/dashboard/templates/performance_report.html
+++ b/distributed/dashboard/templates/performance_report.html
@@ -2,5 +2,5 @@
 {# See https://github.com/bokeh/bokeh/blob/main/bokeh/core/_templates/file.html #}
 
 {% block preamble %}
-<link rel="shortcut icon" href="https://docs.dask.org/en/latest/_static/images/favicon.ico" />
+<link rel="shortcut icon" href="statics/images/favicon.ico" />
 {% endblock %}
